### PR TITLE
App Runner quickstart

### PR DESCRIPTION
**Pull request by 4demos**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/aws/aws-apprunner/config.yml
Comment (jcountsNR) [inline quickstarts/aws/aws-apprunner/config.yml:10]:
```suggestion
icon: logo.png
```

File: quickstarts/aws/aws-apprunner/config.yml
Comment (jcountsNR) [inline quickstarts/aws/aws-apprunner/config.yml:14]:
```suggestion
title: Amazon App Runner
```

These are all with New Relic, for consistency I would just leave the title `Amazon App Runner`

File: quickstarts/aws/aws-apprunner/config.yml
Comment (jcountsNR) [inline quickstarts/aws/aws-apprunner/config.yml:23]:
```suggestion
  - app runner
  - logs
```

the search is case sensitive so I usually leave these lower case, also adding logs will help place this into a category.

**Review by jcountsNR — COMMENTED** [review]
@4demos  , a couple of things on this one and it should be good. 

1. The reference for install plan needs to map to an install plan id, and you will have to provide one (check install/aws/aws-infrastructure-monitoring for an example).
2. you'll need to update the icon to point to png instead of svg
3. The URL for the documentation doesn't seem to work, this won't get approved until then

File: quickstarts/aws/aws-apprunner/config.yml
Comment (jcountsNR) [inline quickstarts/aws/aws-apprunner/config.yml:25]:
```suggestion
  - aws-apprunner
  dashboards:
    - aws-apprunner
```

You'll also want to point to the dashboards for them to display correctly.

**Comment by 4demos** [comment]
> The reference for install plan needs to map to an install plan id, and you will have to provide one (check install/aws/aws-infrastructure-monitoring for an example).
not sure I quite understood how to "provide one". I've created a file install/aws/aws-apprunner/install.yml with id: aws-apprunner (matching one in config.yaml)  but still have the error. Is install/aws/aws-apprunner/install.yml  required for AppRunner case? AppRunner is not an installable package as such. install.yml  just has a reference to the docs.
The final version of the docs should be available from @bradleycamacho

File: quickstarts/aws/aws-apprunner/config.yml
Comment (jcountsNR) [inline quickstarts/aws/aws-apprunner/config.yml:29]:
```suggestion
```

File: quickstarts/aws/aws-apprunner/config.yml
Comment (jcountsNR) [inline quickstarts/aws/aws-apprunner/config.yml:30]:
```suggestion
```

**Comment by jcountsNR** [comment]
@4demos I believe the id is correct, but there is an issue with the white spacing. Try replacing the config.yml with this code and see if it resolves the issue:

`slug: aws-apprunner
description: |
  ## Monitor your Apps running in App Runner coutainers on New Rerilc Instant Observability Platform!
  With New Relic's AWS App Runner integration, you now can see all your App Runner data on New Rerilc Instant Observability Platform.
  ## Get started!
  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-apprunner) to learn more about New 
  Relic monitoring and integration with AWS App Runner.
title: Amazon App Runner
summary: |
  Use New Relic - AWS App Runner Integration to Monitor Your Conteinerized Apps and Services with New Relic
icon: logo.png
level: New Relic
authors:
  - New Relic
documentation:
  - name: Amazon App Runner integration docs
    description: |
      Faster and comprehensive telemetry from Amazon App Runner Service for new and existing New Relic Customers
    url: >-
      https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-apprunner
keywords:
  - aws
  - app runner
  - logs
installPlans:
  - aws-apprunner
dashboards:
  - aws-apprunner`

**Review by jcountsNR — APPROVED** [review]

**Comment by jcountsNR** [comment]
@aswanson-nr  , this one looks good to me. The only issue is that the url does not look like it's live yet.

File: quickstarts/aws/aws-apprunner/config.yml
Comment (aswanson-nr) [inline quickstarts/aws/aws-apprunner/config.yml:6]: This link 404s, might need to get that published before we merge this.

File: quickstarts/aws/aws-apprunner/config.yml
Comment (aswanson-nr) [inline quickstarts/aws/aws-apprunner/config.yml:20]: Same with this url

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (aswanson-nr) [inline dashboards/aws-apprunner/aws-apprunner.json:25]: I noticed these `${}` templates in a few of the NRQL queries, quickstart dashboards do not currently support any type of templating. So I just want to make sure that the full name of the `aws.logGroup` is literally `/aws/apprunner/${AppRunner-container-name}/${application-hash}/application`. This comment applies to a few on the NRQL queries

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (aswanson-nr) [inline dashboards/aws-apprunner/aws-apprunner.json:109]: Will this aws account id be the same for every single user that installs this dashboard? If not, we're going to need to find a way to query for this data without using a specific account id.

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (aswanson-nr) [inline dashboards/aws-apprunner/aws-apprunner.json:130]: Same comment here on account ids

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (aswanson-nr) [inline dashboards/aws-apprunner/aws-apprunner.json:154]: Same comment here on account ids

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (aswanson-nr) [inline dashboards/aws-apprunner/aws-apprunner.json:175]: Same comment here on account ids

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (aswanson-nr) [inline dashboards/aws-apprunner/aws-apprunner.json:199]: Same comment here on account ids

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (aswanson-nr) [inline dashboards/aws-apprunner/aws-apprunner.json:46]: This is another one using the `${}` templating

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (aswanson-nr) [inline dashboards/aws-apprunner/aws-apprunner.json:67]: This is another one using the `${}` templating

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (aswanson-nr) [inline dashboards/aws-apprunner/aws-apprunner.json:88]: This is another one using the `${}` templating

**Review by aswanson-nr — COMMENTED** [review]
Hey @4demos and @jcountsNR, the metadata looks good, but I have a few concerns with the dashboard queries. My comments are attached

**Review by jcountsNR — APPROVED** [review]

**Comment by aswanson-nr** [comment]
Hey @4demos and @jcountsNR, I had a few comments on the NRQL used in the dashboard. If those aren't a concern, then we should be good to merge this in. I'm going to wait for a confirmation from you both before I move anything along.

File: install/aws/aws-apprunner/install.yml
Comment (aswanson-nr) [inline install/aws/aws-apprunner/install.yml:5]:
```suggestion
  AWS Apprunner Containerized Services
```

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (jcountsNR) [inline dashboards/aws-apprunner/aws-apprunner.json:211]:
```suggestion
                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` LIKE '%/application' SINCE 30 minutes ago facet `aws.logGroup` TIMESERIES MAX"
```

Comment (jcountsNR) [reply dashboards/aws-apprunner/aws-apprunner.json:211]: I would not have this query as a hardcoded time, so you can pick some relative time and go with it.

File: quickstarts/aws/aws-apprunner/config.yml
Comment (aswanson-nr) [inline quickstarts/aws/aws-apprunner/config.yml:3]:
```suggestion
  ## Monitor your Apps running in App Runner containers on New Relic Instant Observability Platform!
```

File: quickstarts/aws/aws-apprunner/config.yml
Comment (aswanson-nr) [inline quickstarts/aws/aws-apprunner/config.yml:4]:
```suggestion
  With New Relic's AWS App Runner integration, you now can see all your App Runner data on New Relic Instant Observability Platform.
```

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (jcountsNR) [inline dashboards/aws-apprunner/aws-apprunner.json:265]:
```suggestion
                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` LIKE '%/service' SINCE 30 minutes ago FACET `aws.logGroup` TIMESERIES MAX"
```

File: quickstarts/aws/aws-apprunner/config.yml
Comment (aswanson-nr) [inline quickstarts/aws/aws-apprunner/config.yml:9]:
```suggestion
  Use New Relic - AWS App Runner Integration to Monitor Your Containerized Apps and Services with New Relic
```

File: dashboards/aws-apprunner/aws-apprunner.json
Comment (jcountsNR) [reply dashboards/aws-apprunner/aws-apprunner.json:265]: same issue here.

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @4demos, 
The dashboard updates look good, but I'm still seeing the issue with those doc links and I noticed a few spelling errors.

**Review by jcountsNR — APPROVED** [review]

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**
